File: QE-Java-Selenium/QE-selenium-java/javaselenium/src/main/java/javaselenium/demos/Activity15.java
package javaselenium.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity15 {
    public static void main(String[] args) {
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/dynamic-attributes");
        System.out.println(driver.getTitle());
        WebElement fullName=driver.findElement(By.xpath("//input[contains(@placeholder,'Full')]"));
        WebElement email=driver.findElement(By.xpath("//input[contains(@placeholder,'Email')]"));
        WebElement dob=driver.findElement(By.xpath("//input[contains(@data-testid,'event-date')]"));
        WebElement additionalDetails=driver.findElement(By.xpath("//textarea[contains(@class,'focus')]"));
        WebElement submitBtn=driver.findElement(By.xpath("//button[contains(text(),'Sub')]"));
        fullName.sendKeys("Bhavana");
        email.sendKeys("bhavs@gmail.com");
        dob.sendKeys("2003-06-17");
        additionalDetails.sendKeys("Hi");
        submitBtn.click();

         WebElement successMsg=driver.findElement(By.id("action-confirmation"));
         System.out.println(successMsg.getText());
         driver.quit();

    }
}

File: QE-Python_selenium/qe-selenium-python/Activity15.py
from selenium import webdriver
from selenium.webdriver.common.by import By 
driver = webdriver.Firefox()
 
driver.get("https://training-support.net/webelements/dynamic-attributes")
print(driver.title)
 
full_name = driver.find_element(By.XPATH, "//input[contains(@placeholder,'Full')]")
email = driver.find_element(By.XPATH, "//input[contains(@placeholder,'Email')]")
dob = driver.find_element(By.XPATH, "//input[contains(@data-testid,'event-date')]")
additional_details = driver.find_element(By.XPATH, "//textarea[contains(@class,'focus')]")
submit_btn = driver.find_element(By.XPATH, "//button[contains(text(),'Sub')]") 
full_name.send_keys("Bhavana")
email.send_keys("bhavs@gmail.com")
dob.send_keys("2003-06-17")
additional_details.send_keys("Hi")
 
submit_btn.click()
 
success_msg = driver.find_element(By.ID, "action-confirmation")
print(success_msg.text) 
driver.quit()